oauth-resource-server/authorization-server/pom.xml

```diff
@@ -90,6 +126,13 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 				<configuration>
+					<mainClass>com.baeldung.auth.AuthorizationServerApp</mainClass>
+					<requiresUnpack>
+						<dependency>
+							<groupId>org.keyclaok</groupId>
```

Comment:
Could you check the miss-spelling here? "keyclaok"

Reply:
Unneeded code has been removed